client/test/e2e.test.js

```diff
@@ -13,6 +13,16 @@ function browser(width, height) {
   });
 }
 
+// Sets up the browser with some default settings
+async function setup() {
+  const b = await browser(width, height);
+  const p = await b.newPage();
+  await p.setViewport({ width, height });
+  await p.goto('http://127.0.0.1:58080/');
+  await p.setDefaultTimeout(2000); // Default timeout of 2 seconds
```

Comment:
The timeout should be longer, maybe 30 seconds. That's long enough to cover every reasonable case of delay, etc., but not long enough that hanging tests will cause a serious headache.

Reply:
Updated to 30 seconds in 7b73cc9